### PR TITLE
Various contributor fixes

### PR DESCRIPTION
**Pull request by karsa-mistmere**
## What is the purpose of this pull request?
- [x] Bug fix

### Description
These are manual contributor fixes, adding some contributors who were missing & removing myself from some that were credited automatically but were unwarranted.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/axis-3d.json [hunk 1]
```diff
@@ -2,8 +2,7 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "lscheibel",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```
Comment (danielbayley) [inline icons/axis-3d.json:5]: I don’t recall working on this one at all, either…

File: icons/heading-1.json [hunk 2]
```diff
@@ -2,8 +2,7 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "ericfennis",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```
Comment (danielbayley) [inline icons/heading-1.json:5]: Pretty sure I had nothing to do with this one.

File: icons/heading.json [hunk 3]
```diff
@@ -2,8 +2,7 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "ericfennis",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```
Comment (danielbayley) [inline icons/heading.json:5]: Or this one…

File: icons/signal-low.json [hunk 4]
```diff
@@ -2,8 +2,7 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "ericfennis",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```
Comment (danielbayley) [inline icons/signal-low.json:5]: I wasn’t involved in this one.

File: icons/signal-zero.json [hunk 5]
```diff
@@ -3,7 +3,6 @@
   "contributors": [
     "ericfennis",
     "danielbayley",
```
Comment (danielbayley) [inline icons/signal-zero.json:5]: …or this one.

File: icons/star-half.json [hunk 6]
```diff
@@ -3,8 +3,7 @@
   "contributors": [
     "mittalyashu",
     "ericfennis",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```
Comment (danielbayley) [inline icons/star-half.json:6]: Nor this one.

File: icons/check-check.json [hunk 7]
```diff
@@ -2,7 +2,6 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "ericfennis",
-    "karsa-mistmere",
     "danielbayley"
```
Comment (danielbayley) [inline icons/check-check.json:5]: I wasn’t involved here.

**Review by danielbayley — CHANGES_REQUESTED** [review]

File: icons/axis-3d.json (hunk 1 above)
Comment (karsa-mistmere) [reply icons/axis-3d.json]: I took the liberty to also remove you from these ones.

Comment (danielbayley) [reply icons/axis-3d.json]: > I'm afraid you're probably included in many others as well by mistake

@karsa-mistmere Maybe a few slipped through the net… but I have `push`ed a _lot_ of icons/changes! I’ll just submit another PR to remove if I spot any others going forward…

**Review by danielbayley — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**